_includes/header.html

```diff
@@ -61,30 +61,8 @@ <h1 class="title">
           role="navigation"
           class="navOverlay abs hide list-style-none">
           <li class="p-2 border-b-1">
-            Join a Chapter
+            <a href="/chapters">Join a Chapter</a>
```

Comment:
@rossPatton this here, changes the navigation to be flat/simple link. Could also merge to be at top of the main hamburger menu

Reply:
Right, what i'm suggesting is we just move that link into the hamburger menu, and just remove this link/subnav